data/yam/agama/auto/template.libsonnet

```diff
@@ -31,8 +32,12 @@ function(bootloader=true,
          ssl_certificates=false,
          storage='',
          user=true) {
-  [if bootloader then 'bootloader']: base_lib.stop_timeout(),
-  [if bootloader_timeout then 'bootloader']: base_lib.timeout(),
+  [if bootloader || bootloader_timeout !='' ||
+    bootloader_extra_kernel_params != '' then 'bootloader']: std.prune({
```

Comment:
The bootloader or bootloader_timeout must exist one, so do not need judge the bootloader_extra_kernel_params  here.

Reply:
it is technically correct, but yeah, we don't want to run installation without stopping the timeout or having a big timeout because of the risk of not seeing the grub...